lib/Runtime/Library/BuiltIn/IndexOf.js

```diff
@@ -0,0 +1,48 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+"use strict";
+
+(function arrayIndexOf(searchElement, fromIndex) {
+
+    var k;
+
+    if (this == null) {
+        throw new TypeError('"this" is null or not defined');
+    }
+
+    var o = this;
+
+    var len = o.length >>> 0;
+
+    if (o.length <= 0) {
+        return -1;
+    }
+
+    var n = fromIndex | 0;
```

Comment:
> | 0 [](start = 21, length = 4)

Also use >>>0 here because fromIndex can be as long as the array and still needs to be handled correctly (also as a UInt32).